indexer-app/src/cmr/indexer/data/concepts/collection/keyword.clj

```diff
@@ -1,12 +1,13 @@
 (ns cmr.indexer.data.concepts.collection.keyword
   "Contains functions to create keyword fields"
   (:require
-    [cmr.common.concepts :as concepts]
-    [cmr.common.util :as util]
-    [cmr.indexer.data.concepts.keyword-util :as keyword-util]))
+   [clojure.string :as str]
+   [cmr.common.concepts :as concepts]
+   [cmr.common.util :as util]
+   [cmr.indexer.data.concepts.keyword-util :as keyword-util]))
 
 (defn create-keywords-field
-  [concept-id collection other-fields]
+  [concept-id collection other-fields to-keyword-text?]
   "Create a keyword field for keyword searches by concatenating several other fields
   into a single string"
```

Comment:
The comment should be updated to indicate why one would send true or false for the 3rd parameter.